content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

```diff
@@ -0,0 +1,179 @@
+
+Before starting installation, make sure you've [set up and initialized the Datadog iOS RUM SDK][2].
+
+1. Link both RUM and Session Replay SDKs using git branches in `Podfile` and run `pod install` (or `pod update` if you're upgrading from an existing installation):
+
+   ```ruby
+
+     pod 'DatadogSDK', :git => 'https://github.com/DataDog/dd-sdk-ios.git', :branch => 'session-replay-beta'
+     pod 'DatadogSDKSessionReplay', :git => 'https://github.com/DataDog/dd-sdk-ios.git', :branch => 'session-replay-beta'
+
+   ```
+
+2. To verify installation, check `Podfile.lock`. Both pods should list the same version. For example:
+
+   ```ruby
+     PODS:
+       - DatadogSDK (1.19.0-sr-beta3)
+       - DatadogSDKSessionReplay (1.19.0-sr-beta3):
+         - DatadogSDK (= 1.19.0-sr-beta3)
+   ```
+
+### Installation with Swift Package Manager
+
+Link RUM and Session Replay SDKs by specifying the [`dd-sdk-ios`][3] repository URL and the `session-replay-beta` branch. Ensure that the most recent commit is fetched.
+
+### Enabling Session Replay in the app
+
+To enable Session Replay in the code, import `DatadogSessionReplay` dependency and call `SessionReplay.initialize(with:),` followed by `.start()`. Both must happen after initializing the RUM SDK.
+
+The following snippet illustrates a basic example in `AppDelegate.swift`:
+
+```swift
+import Datadog
+import DatadogSessionReplay
+
+var sessionReplay: SessionReplayController!
+
+// enable RUM SDK:
+Datadog.initialize(
+    appContext: .init(),
+    trackingConsent: .granted,
+    configuration: Datadog.Configuration
+        .builderUsing(
+            rumApplicationID: "<rum-application-id>",
+            clientToken: "<client-token>",
+            environment: "<environment>"
+        )
+        .trackUIKitRUMViews()
+        .trackUIKitRUMActions()
+        .build()
+)
+
+Global.rum = RUMMonitor.initialize()
+
+// enable Session Replay:
+let configuration = SessionReplayConfiguration(privacy: .maskAll)
+sessionReplay = SessionReplay.initialize(with: configuration)
+sessionReplay.start()
+```
+
+[1]: https://github.com/DataDog/dd-sdk-ios/tree/session-replay-beta
+[2]: https://docs.datadoghq.com/real_user_monitoring/ios/?tab=swift
+[3]: https://github.com/DataDog/dd-sdk-ios
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Additional configuration
+
+{{< tabs >}}
+{{% tab "Android" %}}
+### Update the sample rate for recorded sessions to appear
+Beginning with v1.20.0, the recorder sessions are sampled, and the default sample rate is 0 (meaning that no sessions are recorded). To ensure that you have some recorded sessions in your dashboard, you have to explicitly set the desired sample rate in the configuration:
+
+{{< code-block lang="java" filename="build.gradle" disable_copy="false" collapsible="true" >}}
+
+val sessionReplayConfig = SessionReplayConfiguration.Builder()
+ ...
+.setSessionReplaySampleRate([YOUR_SAMPLE_RATE_HERE])
+.build()
+
+{{< /code-block >}}
+
+{{% /tab %}}
+
+{{% tab "iOS" %}}
+
```

Comment:
can we add the snippets here (like we have for android) that have how to adjust the sample rate @ncreated ?